src/kinvey.js

```diff
@@ -184,6 +185,8 @@ Kinvey.AuthorizationGrant = AuthorizationGrant;
 Kinvey.CustomEndpoint = CustomEndpoint;
 Kinvey.DataStore = DataStore;
 Kinvey.DataStoreType = DataStoreType;
+Kinvey.Stream = Stream;
+Kinvey.LiveService = LiveServiceFacade;
```

Comment:
Ok makes sense. I think we should try to only expose `Kinvey.LiveService` and somehow merge `Stream` into this namespace. Seems like this is what you are suggesting.